backend/src/modules/asking-question/core/application/query/FindCurrentGroupQuestionByGroupIdQueryHandler.ts

```diff
@@ -0,0 +1,12 @@
+import { QueryHandler } from '../../../../../shared/core/application/query/QueryHandler';
+import { FindCurrentGroupQuestionByGroupId, FindCurrentGroupQuestionByGroupIdResult } from './FindCurrentGroupQuestionByGroupId';
+import { AskingGroupQuestionRepository } from '../AskingGroupQuestionRepository';
+
+export class FindCurrentGroupQuestionByGroupIdQueryHandler
+  implements QueryHandler<FindCurrentGroupQuestionByGroupId, FindCurrentGroupQuestionByGroupIdResult> {
+  constructor(private readonly repository: AskingGroupQuestionRepository) {}
+
+  execute(query: FindCurrentGroupQuestionByGroupId): Promise<FindCurrentGroupQuestionByGroupIdResult> {
+    return this.repository.findByGroupId(query.groupId);
```

Comment:
Trzymamy w repo tylko jedno pytanie per grupa? Wystarczy to nam? 
Jesli tak to OK. Bo ja np. quizy trzymam stare, i szukajac "current" biore najnowszy.

Reply:
w sumie, może i warto mieć jakąś historię :P dodam